internal/provider/integration_email_resource.go

```diff
@@ -0,0 +1,420 @@
+		return
+	}
+	ctx = tflog.SetField(ctx, "space_mrn", space.MRN())
+
+	// Do GraphQL request to API to create the resource.
+	tflog.Debug(ctx, "Creating integration")
+	integration, err := r.client.CreateIntegration(ctx,
+		space.MRN(),
+		data.Name.ValueString(),
+		mondoov1.ClientIntegrationTypeTicketSystemEmail,
+		mondoov1.ClientIntegrationConfigurationInput{
+			EmailConfigurationOptions: &mondoov1.EmailConfigurationOptionsInput{
+				Recipients:        convertRecipients(data.Recipients),
+				AutoCreateTickets: mondoov1.NewBooleanPtr(mondoov1.Boolean(data.AutoCreateTickets.ValueBool())),
+				AutoCloseTickets:  mondoov1.NewBooleanPtr(mondoov1.Boolean(data.AutoCloseTickets.ValueBool())),
+			},
+		})
+	if err != nil {
+		resp.Diagnostics.
+			AddError("Client Error",
+				fmt.Sprintf("Unable to create Email integration, got error: %s", err),
+			)
+		return
+	}
+
+	// Save space mrn into the Terraform state.
+	data.Mrn = types.StringValue(string(integration.Mrn))
+	data.Name = types.StringValue(string(integration.Name))
+	data.SpaceID = types.StringValue(space.ID())
+
+	// Save data into Terraform state
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+}
+
+func (r *integrationEmailResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	var data integrationEmailResourceModel
+
+	// Read Terraform prior state data into the model
+	resp.Diagnostics.Append(req.State.Get(ctx, &data)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Read API call logic
+
+	// Save updated data into Terraform state
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+}
+
+func (r *integrationEmailResource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	var data integrationEmailResourceModel
+
+	// Read Terraform plan data into the model
+	resp.Diagnostics.Append(req.Plan.Get(ctx, &data)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	// Do GraphQL request to API to update the resource.
+	opts := mondoov1.ClientIntegrationConfigurationInput{
+		EmailConfigurationOptions: &mondoov1.EmailConfigurationOptionsInput{
+			Recipients:        convertRecipients(data.Recipients),
+			AutoCreateTickets: mondoov1.NewBooleanPtr(mondoov1.Boolean(data.AutoCreateTickets.ValueBool())),
+			AutoCloseTickets:  mondoov1.NewBooleanPtr(mondoov1.Boolean(data.AutoCloseTickets.ValueBool())),
+		},
+	}
+
+	_, err := r.client.UpdateIntegration(ctx,
+		data.Mrn.ValueString(),
+		data.Name.ValueString(),
+		mondoov1.ClientIntegrationTypeTicketSystemEmail,
+		opts,
+	)
+	if err != nil {
+		resp.Diagnostics.
+			AddError("Client Error",
+				fmt.Sprintf("Unable to update Email integration, got error: %s", err),
+			)
+		return
+	}
+
+	// Save updated data into Terraform state
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+}
+
+func (r *integrationEmailResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
```

Comment:
Same as https://github.com/mondoohq/terraform-provider-mondoo/pull/167#discussion_r1870371480 please add the import function.